### PR TITLE
[CNI Express]

### PR DESCRIPTION
**Pull request by marciocloudflare**
Closes PCX-9536
Revamps the CNI docs to account for the new CNI Express product

### CODE REVIEW

File: content/network-interconnect/about.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: About
+pcx_content_type: concept
+weight: 2
+meta:
+  title: About Cloudlfare Network Interconnect
```
Comment (ranbel) [inline content/network-interconnect/about.md]:
```suggestion
  title: About Cloudflare Network Interconnect
```

File: content/network-interconnect/about.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# About Cloudflare Network Interconnect
+
+Cloudflare supports a variety of options to connect your network to Cloudflare:
+
+- Express CNI, for Magic WAN and Magic Transit.
+- Classic CNI, for Magic Transit.
+- Cloud CNI, for Magic WAN and Magic Transit.
+- Peering via either an internet exchange, or a private network interconnect (PNI).
```
Comment (ranbel) [inline content/network-interconnect/about.md]:
```suggestion
- Peering via either an Internet exchange, or a private network interconnect (PNI).
```

File: content/network-interconnect/about.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Below is a brief overview of the options to help you decide which method best fits your network.
+
+| Network interconnect mode | Use cases | Capabilities |
+--- | --- | ---
+| [Express CNI](/network-interconnect/express-cni/) | Use for Magic WAN or Magic Transit, interconnecting directly with on-premises locations. |  - 1, 10, 100 Gbps <br>  - Available at Express CNI-capable Cloudflare locations <br>  - No tunnel required <br>  - Set up via dashboard in less than three minutes. |
```
Comment (ranbel) [inline content/network-interconnect/about.md]:
```suggestion
| [Express CNI](/network-interconnect/express-cni/) | Use for Magic WAN or Magic Transit, interconnecting directly with on-premise locations. |  - 1, 10, 100 Gbps <br>  - Available at Express CNI-capable Cloudflare locations <br>  - No tunnel required <br>  - Set up via dashboard in less than three minutes. |
```

File: content/network-interconnect/about.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+| [Classic CNI](/network-interconnect/classic-cni/) | Use for Magic Transit, interconnecting directly or via a partner with on-premises locations. |  - 10, 100 Gbps <br>  - Available at classic CNI capable Cloudflare edge locations <br>  - Overlay GRE tunnel required for egress <br>  - BGP route reflector signaling <br>  - Setup facilitated by account team. |
```
Comment (ranbel) [inline content/network-interconnect/about.md]:
```suggestion
| [Classic CNI](/network-interconnect/classic-cni/) | Use for Magic Transit, interconnecting directly or via a partner with on-premise locations. |  - 10, 100 Gbps <br>  - Available at classic CNI capable Cloudflare edge locations <br>  - Overlay GRE tunnel required for egress <br>  - BGP route reflector signaling <br>  - Setup facilitated by account team. |
```

Comment (ranbel) [reply content/network-interconnect/about.md]: "Cloudflare edge locations" --> "Cloudflare data centers"?

File: content/network-interconnect/about.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+| [Cloud CNI](/network-interconnect/cloud-cni/) | Use for Magic WAN or Magic Transit, interconnecting directly with virtual networks (VPCs) from public cloud providers. |  - Speed depends on cloud provider <br>  - Available at Cloud CNI capable Cloudflare locations <br>  - Setup facilitated by account team. |
+| [Peering, or private network interconnect (PNI)](/network-interconnect/pni-and-peering/) | Use for connecting your users to any Cloudflare services, via a more direct, performant and potentially cost-effective network path. | - No requirement to be a customer <br> - Speed depends on IX or PNI <br> - Available at PNI capable Cloudflare locations (PNI) and any Internet Exchange (IX) where Cloudflare peer today. <br>  - Setup facilitated by `peering@cloudflare.com` |
```
Comment (ranbel) [inline content/network-interconnect/about.md]:
```suggestion
| [Peering, or private network interconnect (PNI)](/network-interconnect/pni-and-peering/) | Use for connecting your users to any Cloudflare services, via a more direct, performant and potentially cost-effective network path. | - No requirement to be a customer <br> - Speed depends on IX or PNI <br> - Available at PNI capable Cloudflare locations (PNI) and any Internet Exchange (IX) where Cloudflare peers today. <br>  - Setup facilitated by `peering@cloudflare.com` |
```

Comment (ranbel) [reply content/network-interconnect/about.md]: Is "peer" supposed to be a verb?

File: content/network-interconnect/classic-cni/_index.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Cloudflare supports two classic CNI connectivity options: direct and partner. The network interconnect is the link between your infrastructure and Cloudflare's network.
+
+The use case for classic CNI is Magic Transit. If your origins are behind Magic Transit over a classic CNI, then all Cloudflare services that work with public origins will run over the CNI (for example Load Balancer, WAF, Cache, etc).
+
+## Classic CNI (Direct)
+
+Direct connections are available at any of our [private peering facilities](https://www.peeringdb.com/net/4224).
+
+Choose a direct connection if you are co-located with a Cloudflare data center. We also recommend a direct connect if:
+- Your link size is 10G or higher
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/_index.md:20]: Should the number be 10Gbps as described on the About page?

File: content/network-interconnect/classic-cni/_index.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- You want higher throughput than a partner connection
+- You want to eliminate as many intermediaries from an interconnect as possible
+
+### Limitations
+
+Classic CNI (Direct) supports 10G or 100G interfaces.
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/_index.md:26]: same question as above for 10G and 100G

File: content/network-interconnect/classic-cni/set-up/configure-bgp-bfd.md [hunk 8]
```diff
@@ -21,24 +27,30 @@ Cloudflare v6: 2001:db8:12:3::7ac2:d64a/127
...
-## Go live
+### Go live
+
+When traffic begins flowing over the connection, you are fully set up with CNI.
+
+## BFD
+
+Bidirectional Forwarding Detection (BFD) is a networking protocol that constantly monitors links and BGP sessions down to the second by sending a constant stream of traffic across the session.
+
+If a small number of packets does not make it to the other side of the session, the session is considered down. This solution is useful for users who cannot tolerate any amount of packet loss during a session.
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/set-up/configure-bgp-bfd.md]:
```suggestion
If a small number of packets do not make it to the other side of the session, the session is considered down. This solution is useful for users who cannot tolerate any amount of packet loss during a session.
```

File: content/network-interconnect/classic-cni/set-up/configure-bgp-bfd.md [hunk 9]
```diff
@@ -21,24 +27,30 @@ Cloudflare v6: 2001:db8:12:3::7ac2:d64a/127
...
 
-When traffic begins flowing over the connection, you are fully set up with CNI.
\ No newline at end of file
+Bidirectional Forwarding Detection is only supported for users with direct connections. To begin using BFD, contact your Implementation Manager.
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/set-up/configure-bgp-bfd.md]: Should this say "contact your account team"?

File: content/network-interconnect/classic-cni/set-up/partners/console-connect.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+# Console Connect Users
+
+1. Select **+** to create a new connection.
+2. From the **Choose source port** dropdown, choose your source port.
+3. Select **Another Company**, and then enter **Cloudflare**.
+4. For **Rate Limit (speed)**, choose your rate limit.
+5. Select **Next**.
+6. Review the pricing information, and when you are done, select **Save Payment**.
+7. Cloudflare will accept the connection and your Account Team will get in touch with next steps for BGP session establishment.
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/set-up/partners/console-connect.md]:
```suggestion
7. Cloudflare will accept the connection and your account team will get in touch with next steps for BGP session establishment.
```

File: content/network-interconnect/classic-cni/set-up/partners/equinix-fabric.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+weight: 3
+meta:
+  title: Equinix Users
+---
+
+# Equinix Users
+
+## Order an Equinix Fabric Connection
+
+After you confirm the locations and speed with you account team, order the Equinix Fabric connections to Cloudflare.
```
Comment (ranbel) [inline content/network-interconnect/classic-cni/set-up/partners/equinix-fabric.md]:
```suggestion
After you confirm the locations and speed with your account team, order the Equinix Fabric connections to Cloudflare.
```

File: content/network-interconnect/cloud-cni.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+- Amazon Web Services (AWS) DirectConnect
+- Google Cloud (GCP) Interconnect
+- IBM Cloud Direct Link
+- Oracle Cloud Infrastructure (OCI) FastConnect
+
+Azure ExpressRoute support is coming soon.
+
+## Cloud CNI Setup
+
+Enterprise customers using Magic WAN and Magic Transit can get started with Cloud CNI by contacting their customer success manager.
```
Comment (ranbel) [inline content/network-interconnect/cloud-cni.md]:
```suggestion
Enterprise customers using Magic WAN and Magic Transit can get started with Cloud CNI by contacting their account team.
```

File: content/network-interconnect/express-cni/create-interconnects.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+3. In **Types of CNIs** > **Direct**, select **Create new**.
+4. In **Enter your location**, search for the location where you have the data centers you want to connect to Cloudflare through Express CNI.
+5. From the list of locations, choose the one that suits you with **Select the location**.
+
+    {{<Aside type="note">}}If your location does not come up in the search, that means that Express CNI is not available in your location yet. Refer to [Classic CNI](/network-interconnect/classic-cni/) to learn how you can use the classic version of network interconnect to connect your location to Cloudflare.{{</Aside>}}
+
+6. In **Choose your interconnect speed**, select either **1 Gigabit** or **10 Gigabit**.
+7. After choosing a speed, give your interface a name and select **Continue**.
+
+    {{<Aside type="note">}}Contact your implementation manager if you need 100 Gigabit connections.{{</Aside>}}
```
Comment (ranbel) [inline content/network-interconnect/express-cni/create-interconnects.md]:
```suggestion
    {{<Aside type="note">}}Contact your account team if you need 100 Gigabit connections.{{</Aside>}}
```

**Review by ranbel — APPROVED** [review]

File: content/network-interconnect/about.md (hunk 4 above)
Comment (marciocloudflare) [reply content/network-interconnect/about.md]: Yes! Good catch :)

File: content/network-interconnect/about.md (hunk 5 above)
Comment (marciocloudflare) [reply content/network-interconnect/about.md]: Not sure. I'll have a think about this

**Pull request merged**